app/Models/User.php

```diff
@@ -42,5 +45,11 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
         'password' => 'hashed',
+        'role_id' => RoleName::class,
```

Comment:
I am not sure if this will work since `role_id` is a foreign key in the database and it can be different in different environments. If it was a static value like status or type, it would work. It can only be integer in this case.

Reply:
I ran the migrations and seeds and it worked fine